### PR TITLE
[on][crit] new option

### PR DESCRIPTION
**Pull request by bw1**
@dgl

### CODE REVIEW

**Comment by ailin-nemui** [comment]
it would be better if you can keep the old format, for backward compatibility

**Comment by bw1** [comment]
### old:
```
/on add 332 /echo hallo welt
/on add 332 -perl print "hallo perl";
/on add 332 -server /echo hallo welt
/on add 332 -channel #test /echo hallo welt
/on add 332 -stop /echo hallo welt
/on add 332 -perl -stop print "hallo perl";
```

### on.save
```
event 332^@all^@^@^@/echo hallo welt
event 332^@all^@1^@^@print "hallo perl";
event 332^@mylocalnet^@^@^@/echo hallo welt
event 332^@all^@^@#test^@/echo hallo welt
event 332^@all^@1^@^@/echo hallo welt
event 332^@mylocalnet^@1^@^@/echo hallo welt
event 332^@all^@11^@^@print "hallo perl";
```

-stop and -perl is the same in the file

File: scripts/on.pl [hunk 1]
```diff
@@ -138,8 +133,11 @@ sub signal_handler {
    my($item, @stuff) = @_;
    my $signal = Irssi::signal_get_emitted();
 
+
    if(exists $on{$signal}) {
       for(@{$on{$signal}}) {
+         debug(Dumper($_));
```
Comment (ailin-nemui) [inline scripts/on.pl]: I think there is error here...

**Comment by bw1** [comment]
:-/

done

**Pull request merged**